UMLEditor/UMLEditor/Views/RelationshipLine.cs

```diff
@@ -70,33 +77,89 @@ public RelationshipLine(UserControl startCtrl, UserControl endCtrl, string relat
         Segments = new List<Line>();
     }
 
```

Comment:
(Note for line 49 since Github isn't letting me make a comment up there). Symbol is not nullable but is not being initialized in the constructor. We should either make it nullable or find a way to assign it on construction to address these warnings.